homebrew_releaser/formula.py

```diff
@@ -204,11 +215,11 @@ def install
             'autogenerated_tar_checksum': autogenerated_tar_checksum,
             'license_type': license_type,
             'dependencies': dependencies_list,
```

Comment:
question: `depends_on` could be multi-line too, do we need to wrap `dependencies_list` in this new function too? Are there others we'd need/want to do the same for?

Reply:
As I read `Formula.generate_formula_data`, the elements of `dependencies_list` can never be multi-line, because `depends_on` is split by line when the list is constructed:
https://github.com/Justintime50/homebrew-releaser/blob/d5b309dbd680fed13895f81c09b4e45bbd2d410a/homebrew_releaser/formula.py#L71-L77

And, then, the list is rendered as a `dependencies` Mustache section, which will loop for each list element:
https://github.com/Justintime50/homebrew-releaser/blob/d5b309dbd680fed13895f81c09b4e45bbd2d410a/homebrew_releaser/formula.py#L139-L141

Is there something I missed where the `{{{dependency}}}` item in that section could end up having line breaks in its string data?

Reply:
My apologies, it's been awhile since I built this piece and forgot how it worked. Users specify their dependencies as a multi-line string; however, we'll construct individual newline `depends_on` entries for each one they passed us so the matching indents shouldn't matter since we'll be doing that for them anyway. So this particular point of feedback isn't valid.